dobble/__main__.py

```diff
@@ -22,6 +22,7 @@ def main(symbols_folder: str,
          mask_ths: int = 250,
```

Comment:
Thanks @BijanRafraf !
I'd suggest to instead add a binary arg indicating if we want the normal or junior version
that way we can control the possible values of `n_symbols_per_card`